### PR TITLE
[Feature] Invite Friends screen updates (#897)

### PR DESCRIPTION
**Pull request by AdnanZiSAP**
## What this PR do

Updates Invite Friends screen:

- New illustration added.
- Subtitle/subtext added.
- Texts updates.
- Styles made consistent.

## The looks

IMG_FEC1D4216913-1

IMG_B73B9A0F5206-1

IMG_B3E7C29AE470-1

IMG_2B95C12BEA85-1

### CODE REVIEW

**Review by haosap — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -417,11 +417,12 @@ Tuis id ullamcorper lectus, feugiat dapibus lectus. In hac habitasse platea dict
...
-"InviteFriends_Description" = "Je mehr Menschen mitmachen, desto flacher bekommen wir die Kurve. Laden Sie Familie, Freunde und Bekannte zum Downloaden ein!";
-"InviteFriends_Button" = "App teilen";
-"InviteFriends_ShareTitle" = "Zusammen gegen Corona. Ich bin dabei. Du auch?";
+"InviteFriends_NavTitle" = "Teilen";
+"InviteFriends_Subtitle" = "Teilen Sie die Corona-Warn-App";
+"InviteFriends_Title" = "Gemeinsam Corona bekämpfen";
+"InviteFriends_Description" = "Je mehr Menschen mitmachen, desto besser durchbrechen wir Infektionsketten. Laden Sie Familie, Freunde und Bekannte ein!";
+"InviteFriends_Button" = "Download-Link versenden";
+"InviteFriends_ShareTitle" = "Gemeinsam Corona bekämpfen. Ich bin dabei. Du auch?";
 "InviteFriends_ShareUrl" = "https://www.corona-warn-app.de";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:426]: Is this link still valid?

Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:426]: https://www.coronawarn.app/ please change to that url

**Review by steins-code — CHANGES_REQUESTED** [review]
Please change the URL as described in user story 737

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:426]: Where's that coming from?

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -417,12 +417,13 @@ Tuis id ullamcorper lectus, feugiat dapibus lectus. In hac habitasse platea dict
...
-"InviteFriends_Button" = "App teilen";
-"InviteFriends_ShareTitle" = "Zusammen gegen Corona. Ich bin dabei. Du auch?";
-"InviteFriends_ShareUrl" = "https://www.corona-warn-app.de";
+"InviteFriends_NavTitle" = "Teilen";
+"InviteFriends_Subtitle" = "Teilen Sie die Corona-Warn-App";
+"InviteFriends_Title" = "Gemeinsam Corona bekämpfen";
+"InviteFriends_Description" = "Je mehr Menschen mitmachen, desto besser durchbrechen wir Infektionsketten. Laden Sie Familie, Freunde und Bekannte ein!";
+"InviteFriends_Button" = "Download-Link versenden";
+"InviteFriends_ShareTitle" = "Gemeinsam Corona bekämpfen. Ich bin dabei. Du auch?";
+"InviteFriends_ShareUrl" = "https://www.coronawarn.app/en/";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:426]: Guess this one is the en version but we need the de version in here :)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: Done [here](https://github.com/corona-warn-app/cwa-app-ios/pull/141/commits/aef5bd26ac6bb4a1cfd9f96a9e1fde00a03ca812). But I would still like to have answer on the question I asked. :)

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:426]: Ok, will change.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: Done again, [here](https://github.com/corona-warn-app/cwa-app-ios/pull/141/commits/57a1db37f672db6bd2188c4690f4eae5888eb4dd). Now both DE and EN.

**Comment by AdnanZiSAP** [comment]
> Please change the URL as described in user story 737



> Please change the URL as described in user story 737

Did.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 2 above)
Comment (AdnanZiSAP) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: Done [here](https://github.com/corona-warn-app/cwa-app-ios/pull/141#pullrequestreview-422692035)

**Review by steins-code — APPROVED** [review]

**Pull request merged**